yb-voyager/src/query/queryissue/issues_ddl_test.go

```diff
@@ -273,6 +274,19 @@ func testSecurityInvokerView(t *testing.T) {
 	assertErrorCorrectlyThrownForIssueForYBVersion(t, err, "unrecognized parameter", securityInvokerViewIssue)
 }
 
+func testForeignKeyReferencesPartitionedTableIssue(t *testing.T) {
+	ctx := context.Background()
+	conn, err := getConn()
```

Comment:
Consider using the new testcontainers package in this test file.

Reply:
Can we do it separately for these issues integration tests as it will require changes to all these tests to not use the connection string to run the SQL commands directly and assert the error instead use the ExecuteSqls of the testContainer